tsconfig.json

```diff
@@ -12,12 +12,11 @@
     "importHelpers": true,
     "target": "es2015",
     "module": "es2020",
-    "lib": [
-      "es2018",
-      "dom"
-    ]
+    "lib": ["es2018", "dom"]
   },
   "angularCompilerOptions": {
-    "enableI18nLegacyMessageIdFormat": false
+    "enableI18nLegacyMessageIdFormat": false,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true
```

Comment:
@namita-25 - ' "allowSyntheticDefaultImports": true,
    "esModuleInterop": true'  

is it required? why?

Reply:
swaggar-ui library 3.18.3 while import in TS component file it shows error and can not see swaggar UI as per requirement (shows blank page)
After adding 
"allowSyntheticDefaultImports": true,
"esModuleInterop": true'

in tsconfig.json file error goes and display swaggar ui in browser